medcat/meta_cat.py

```diff
@@ -370,8 +406,11 @@ def load(cls, save_dir_path: str, config_dict: Optional[Dict] = None) -> "MetaCA
             logger.warning('Loading a MetaCAT model without GPU availability, stored config used GPU')
             config.general['device'] = 'cpu'
             device = torch.device('cpu')
-        meta_cat.model.load_state_dict(torch.load(model_save_path, map_location=device))
 
+        try:
+            meta_cat.model.load_state_dict(torch.load(model_save_path, map_location=device))
+        except:
```

Comment:
Can we be more specific on the exception caught, and pass the exception into the log message at least.